crates/swc_plugin_proxy/src/source_map/plugin_source_map_proxy.rs

```diff
@@ -9,6 +9,16 @@ use crate::memory_interop::read_returned_result_from_host_fallible;
 #[cfg(target_arch = "wasm32")]
 extern "C" {
     fn __lookup_char_pos_source_map_proxy(byte_pos: u32, allocated_ret_ptr: i32) -> i32;
+    fn __doctest_offset_line_proxy(orig: u32) -> u32;
+    fn __merge_spans_proxy(
```

Comment:
I choose to bypass serialization by pass all of the internal values of `Span` with tradeoff of having too many params ™️